gradle.properties

```diff
@@ -0,0 +1,4 @@
+publishConfig.name=GitHubPackages
+publishConfig.url=https://maven.pkg.github.com/nasa-ammos/aerie
+publishConfig.usernameEnvironmentVariable=GITHUB_ACTOR
+publishConfig.passwordEnvironmentVariable=GITHUB_TOKEN
```

Comment:
For brevity I'd suggest changing the prefixes here to just `publish.*` or `publishing.*`.

Reply:
Done, changed it to `publishing`.